### PR TITLE
speaker names in sermons list

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -4,4 +4,13 @@ module.exports = withPWA({
 	pwa: {
 		dest: 'public',
 	},
+	async redirects() {
```
Comment (mattleff) [inline next.config.js:7]: Does this need to be `async`?

File: src/containers/testimonies.module.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @import '../styles/common';
 
-.list {
+.table {
```
Comment (mattleff) [inline src/containers/testimonies.module.scss:3]: This shouldn't be renamed...

File: src/components/molecules/recordingListEntry.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+				</a>
+			</td>
+			<td>
+				<a href={`/${lang}/sermons/${sermon.id}`}>{sermon.title}</a>
+			</td>
+			<td className={styles.presenters}>
+				<ul>
+					{persons.map((p: Person, i: number) => {
+						return (
+							<li key={i}>
```
Comment (mattleff) [inline src/components/molecules/recordingListEntry.tsx:33]: You could use `p.id` here since that should be unique.

**Review by mattleff — APPROVED** [review]
LGTM! 🧑‍🦱

File: next.config.js (hunk 1 above)
Comment (narthur) [reply next.config.js:7]: No, but I think I'll leave this for now so I know how it should work once we start doing dynamic redirects.

**Pull request merged**